### PR TITLE
Update Github issue template [skip ci]

### LINKED ISSUE #3263: Update github issue template to be current
At least remove the reference to a non-existent slack channel.

### PR DESCRIPTION
**Pull request by anchal-agrawal**
This change adds helpful information to the issue template.

/cc @mdubya66 

Fixes #3263

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,26 +1,32 @@
 <!--
-Thank you for contributing to VIC! Please review this checklist
-before submitting your issue.
+Thank you for contributing to VIC! Please review this checklist before submitting your issue.
```
Comment (mdubya66) [inline .github/ISSUE_TEMPLATE.md]: Change this to VIC Engine

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,26 +1,32 @@
...
 
 [ ] Search for duplicates before logging new issues
     https://github.com/vmware/vic/issues?q=is%3Aopen+is%3Aissue
-[ ] Questions are best asked and answered at Slack
-    https://vmware.slack.com/messages/wolverines/
+[ ] Questions are best asked and answered on Slack
+    Internal channel: https://vmware.slack.com/messages/vic-dev
```
Comment (mdubya66) [inline .github/ISSUE_TEMPLATE.md]: remove the internal channel

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -1,26 +1,32 @@
...
 
 **Expected behavior:**
 
-**Actual behavior:**
+**Logs:**
+
+* If you can access the VCH Admin portal, please download and attach the log bundle(s). See https://vmware.github.io/vic/assets/files/html/vic_admin/log_bundles.html for details.
+* If the VCH Admin portal is inaccessible, you can enable SSH to the VCH endpoint VM to obtain logs manually. See https://vmware.github.io/vic/assets/files/html/vic_admin/vch_ssh_access.html for details. The VCH logs will be under /var/log/vic/ on the VM.
+
+**Additional details as necessary:**
```
Comment (mdubya66) [inline .github/ISSUE_TEMPLATE.md]: lgtm with the above changes

**Comment by mdubya66** [comment]
lgtm with changes mentioned in review

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (anchal-agrawal) [inline .github/ISSUE_TEMPLATE.md]: Fixed.

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (anchal-agrawal) [inline .github/ISSUE_TEMPLATE.md]: Done.

**Comment by rajanashok** [comment]
LGTM

**Pull request merged**